packages/react-components/src/components/LocalVideoCameraButton.tsx

```diff
@@ -3,26 +3,26 @@
 
 import { IconButton } from '@fluentui/react';
 import React from 'react';
-import { OptionsDevice } from '../DevicesButton';
-import { localVideoCameraCycleButtonStyles } from '../styles/VideoGallery.styles';
+import { OptionsDevice } from './DevicesButton';
+import { localVideoCameraCycleButtonStyles } from './styles/VideoGallery.styles';
 
 /**
  * @internal
  */
 export interface LocalVideoCameraCycleButtonProps {
   /** Array of cameras available to the user. */
-  cameras: OptionsDevice[];
+  cameras?: OptionsDevice[];
   /** Currently selected camera in the local video stream. */
-  selectedCamera: OptionsDevice;
+  selectedCamera?: OptionsDevice;
   /** callback function to change video feed. */
-  onSelectCamera: (device: OptionsDevice) => Promise<void>;
+  onSelectCamera?: (device: OptionsDevice) => Promise<void>;
```

Comment:
Should these all be required? 

This is an internal API, so we know that these props will always be set. So tell typescript to yell if we do it wrong.

Reply:
the way useHandlers is implemented does not like the guarantee that there is a callback there. 😥